jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/CommandlineTest.java

```diff
@@ -13,77 +13,74 @@
  */
 package org.eclipse.jkube.kit.common.util;
 
-import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Stream;
 
-import static org.junit.Assert.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
-public class CommandlineTest {
+class CommandlineTest {
 
     List<String> result = new ArrayList<>();
     List<String> expected = new ArrayList<>();
```

Comment:
These fields are not OK. If we were to run these tests in parallel, they'd fail.
In addition, the lists might be polluted, or not clean after a given test execution